Modules/Loadable/Volumes/Widgets/qSlicerScalarVolumeDisplayWidget.cxx

```diff
@@ -241,7 +241,10 @@ void qSlicerScalarVolumeDisplayWidget::updateHistogram()
   vtkDataArray* voxelValues = pointData ? pointData->GetScalars() : nullptr;
 
   // If there are no voxel values then we completely hide the histogram section
```

Comment:
```suggestion
  // Guard against calling setVisible() too early.
  //
  // ctkCollapsibleGroupBox ignores child visibility changes until it has
  // been created (WA_WState_Created is true). Calling setVisible() before
  // this point can cause its internal visibility bookkeeping to become
  // inconsistent, leaving children in the wrong state.
  //
  // See ctkCollapsibleGroupBoxPrivate::setChildVisibility() for details.
```

Reply:
I've pushed the update with this comment